sql/core/src/test/resources/sql-tests/results/ansi/string-functions.sql.out

```diff
@@ -824,3 +824,116 @@ select to_number('00,454.8-', '00,000.9-')
 struct<to_number(00,454.8-, 00,000.9-):decimal(6,1)>
 -- !query output
 -454.8
+
+
+-- !query
+select to_binary('abc')
+-- !query schema
+struct<to_binary(abc):binary>
+-- !query output
+� 
+
+
+-- !query
+select to_binary('abc', 'utf-8')
+-- !query schema
+struct<to_binary(abc, utf-8):binary>
+-- !query output
+abc
+
+
+-- !query
+select to_binary('abc', 'base64')
+-- !query schema
+struct<to_binary(abc, base64):binary>
+-- !query output
+i�
+
+
+-- !query
+select to_binary('abc', 'base2')
```

Comment:
Should this be an error instead? For base2 it is expecting a string of 0s and 1s.

Reply:
Please correct me if I'm wrong:

My understanding is `base2`(`binary`) should use the default encoding/decoding format, it happens to be `utf-8` here.

We may also exclude `base2`, following what Snowflake [to_binary](https://docs.snowflake.com/en/sql-reference/functions/to_binary.html) does.

Also CC @cloud-fan @HyukjinKwon

Reply:
let's exclude `base2` as its behavior is a bit arguable.